src/wasm/wasm-emscripten.cpp

```diff
@@ -627,6 +634,24 @@ Const* AsmConstWalker::resolveConstAddr(Expression* arg, const Name& target) {
   return arg->cast<Const>();
 }
 
+Name* AsmConstWalker::resolveGlobalGet(Expression* arg, const Name& target) {
```

Comment:
It would be nice to have a comment illustrating the code pattern we are processing here and where/why it shows up.